### PR TITLE
FileOrData: Handle None object

### PR DESCRIPTION
**Pull request by Akasurde**
Return when object is None in FileOrData class

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

**Comment by Akasurde** [comment]
@roycaihw Does this require anything from my side?

**Comment by Akasurde** [comment]
@roycaihw @palnabarun Any news? Do you need anything my side? Thanks.

**Comment by Akasurde** [comment]
@roycaihw @palnabarun Any news ?

File: config/kube_config.py [hunk 1]
```diff
@@ -97,6 +97,8 @@ def __init__(self, obj, file_key_name, data_key_name=None,
         self._file = None
         self._data = None
         self._base64_file_content = base64_file_content
+        if not obj:
+            return
```
Comment (yliaog) [inline config/kube_config.py:101]: the change is fine by itself. however, curious to know what motivates this change? obj is not going to be None in any call site in this repo.

Comment (Akasurde) [reply config/kube_config.py:101]: @yliaog Thanks for the feedback. When there is a typo in kubeconfig it set obj to None, please check this https://github.com/ansible-collections/community.kubernetes/issues/73#issuecomment-653140432

**Comment by yliaog** [comment]
Thanks.

/lgtm
/approve

**Comment by Akasurde** [comment]
@yliaog Thanks for the review and merge.

**Pull request merged**